gems/aws-sdk-transcribestreamingservice/aws-sdk-transcribestreamingservice.gemspec

```diff
@@ -0,0 +1,29 @@
+# WARNING ABOUT GENERATED CODE
+#
+# This file is generated. See the contributing guide for more information:
+# https://github.com/aws/aws-sdk-ruby/blob/master/CONTRIBUTING.md
+#
+# WARNING ABOUT GENERATED CODE
+
+Gem::Specification.new do |spec|
+
+  spec.name          = 'aws-sdk-transcribestreamingservice'
+  spec.version       = File.read(File.expand_path('../VERSION', __FILE__)).strip
+  spec.summary       = 'AWS SDK for Ruby - Amazon Transcribe Streaming Service'
+  spec.description   = 'Official AWS Ruby gem for Amazon Transcribe Streaming Service. This gem is part of the AWS SDK for Ruby.'
+  spec.author        = 'Amazon Web Services'
+  spec.homepage      = 'http://github.com/aws/aws-sdk-ruby'
+  spec.license       = 'Apache-2.0'
+  spec.email         = ['trevrowe@amazon.com']
+  spec.require_paths = ['lib']
+  spec.files         = Dir['lib/**/*.rb']
+
+  spec.metadata = {
+    'source_code_uri' => 'https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-transcribestreamingservice',
+    'changelog_uri'   => 'https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-transcribestreamingservice/CHANGELOG.md'
+  }
+
+  spec.add_dependency('aws-sdk-core', '~> 3', '>= 3.44.1')
+  spec.add_dependency('aws-sigv4', '~> 1.1')
+
```

Comment:
Would this also be dependent on the sigv4 changes where it pulls in an eventstream dependency from #1946?